src/coffee/services/s3client.coffee

```diff
@@ -220,10 +220,13 @@ class S3Client
           Compress.compressImage(tmp_resized, tmpDir, extension)
       .then =>
         @sendMetrics 'increment', 'image.resized'
-        header = 'x-amz-acl': 'public-read'
+        headers = 'x-amz-acl': 'public-read'
+        if !isNaN(expire)
+          headers['Cache-Control'] = 'max-age=' + expire + ', public'
+          headers['Expires'] = 'Sun, 01 Jan 2034 00:00:00 GMT'
```

Comment:
There was one source which suggested to set both Cache-Control and Expires.
C-C takes precedence over Expires.
So this Expires header was set ‘just in case’ and it is safe date.
It was pretty old comment, when C-C wasn't that long on the market. 
Right now using C-C should be enough for this purposes. 
I will remove Expires header.